python/ray/_private/runtime_env/container.py

```diff
@@ -33,10 +34,29 @@ async def setup(
             "--network=host",
             "--pid=host",
             "--ipc=host",
-            "--env-host",
+            # NOTE(zcin): Mounted volumes in rootless containers are
+            # owned by the user `root`. The user on host (which will
+            # usually be `ray` if this is being run in a ray docker
+            # image) who started the container is mapped using user
+            # namespaces to the user `root` in a rootless container. In
+            # order for the Ray Python worker to access the mounted ray
+            # tmp dir, we need to use keep-id mode which maps the user
+            # as itself (instead of as `root`) into the container.
+            "--userns=keep-id",
         ]
+
         container_command.append("--env")
         container_command.append("RAY_RAYLET_PID=" + os.getenv("RAY_RAYLET_PID"))
+        container_command.append("--env")
+        container_command.append("RAY_JOB_ID=$RAY_JOB_ID")
+        for env_var_name, env_var_value in os.environ.items():
```

Comment:
So, we are syncing with all env vars starting with RAY_ right? 

Can you add an unit test for this?